PortalFetch/crawler_data_process.py

```diff
@@ -1,68 +1,167 @@
 #!/usr/bin/env python
-"""Process html course information to save in a .csv file.
+"""Process html course information to save in a .json file.
 
 It requires file 'user.ini' to load the user's own user name and password.
 """
 from bs4 import BeautifulSoup
-import csv
-import logging
+import json
+import time
+import os
 
 
 class DataProcess:
-    """Deputy HTML text to save in a .csv file."""
+    """Deputy .html file to .json file.
```

Comment:
nit: in the future, it could be better to add some unit test for the code. don't have to be very complicated unit test, some simple tests are enough.